CONTRIBUTORS.yaml

```diff
@@ -216,6 +216,8 @@ joachimwolff:
 
 jpmaree:
 
+jraysajulga:
```

Comment:
@jraysajulga Thanks for contributing to this training material! We added you to our [Hall of Fame](https://galaxyproject.github.io/training-material/hall-of-fame), please add any information here you want (name, contact details)

ping @subinamehta

Reply:
thanks!